src/components/activitylog.js

```diff
@@ -5,13 +5,14 @@ define(["events", "globalize", "dom", "datetime", "userSettings", "serverNotific
         var html = "";
         html += '<div class="listItem listItem-border">';
         var color = "Error" == entry.Severity || "Fatal" == entry.Severity || "Warn" == entry.Severity ? "#cc0000" : "#00a4dc";
+        var icon = "Error" == entry.Severity || "Fatal" == entry.Severity || "Warn" == entry.Severity ? "notification_important" : "notifications";
```

Comment:
Since these two lines use the same logic, it may be cleaner to use an `if` statement rather than duplicating it.

Reply:
That's right, hopefully better now